Core/GraphQL/GraphQLFiles/sell_getFulfilmentPoints.graphql

```diff
@@ -1,6 +1,8 @@
 query getFulfilmentPoints($pageSize: Int!, $page: Int = 1, $filters: FulfilmentPointFilter = null) {
   getFulfilmentPoints(page: $page, pageSize: $pageSize, filters: $filters) {
-    edges { ...FragmentFulfilmentPoint }
+    edges { 
+      ...FragmentFulfilmentPoint 
+      }
```

Comment:
Could you resolve the indentation please?

- [ ] re-indent this line

Reply:
Yes, thank you.